libs/auth-api-lib/seeders/data/scope-iceland-health.ts

```diff
@@ -0,0 +1,12 @@
+import { createScope } from './helpers'
+
+export const up = createScope({
+  name: '@island.is/iceland-health',
+  displayName: 'Iceland Health applications',
+  description: 'Allow access to Iceland Health applications',
```

Comment:
Unfortunately the seed helper does not support creating translations, or adding scopes to folders.

Let's keep these in Icelandic to start and after the release you can ask on #ids_support to add the English translation, and add the scope to the Applications group/folder.